src/port/port.tcl

```diff
@@ -4495,6 +4499,9 @@ proc parse_options { action ui_options_name global_options_name } {
                     set kargc [lindex $kopts 0 1]
                     if {$kargc == 0} {
                         set global_options(ports_${action}_${key}) yes
+                        if {${action} eq "info"} {
+                            lappend options_info_order ports_${action}_${key}
+                        }
```

Comment:
I am not particularly fond of special cases for a certain action in the parser. I would suggest to always store `options_${action}_order` to keep this generic. Maybe the order will also be useful for other actions later.